tasmota/xsns_13_ina219.ino

```diff
@@ -85,67 +92,92 @@
 #define INA219_REG_CURRENT                      (0x04)
 #define INA219_REG_CALIBRATION                  (0x05)
 
+#define INA219_DEFAULT_SHUNT_RESISTOR_MILLIOHMS (100.0) // 0.1 Ohm
+
 uint8_t ina219_type[4] = {0,0,0,0};
 uint8_t ina219_addresses[] = { INA219_ADDRESS1, INA219_ADDRESS2, INA219_ADDRESS3, INA219_ADDRESS4 };
 
-uint32_t ina219_cal_value = 0;
-// The following multiplier is used to convert raw current values to mA, taking into account the current config settings
-uint32_t ina219_current_divider_ma = 0;
+#define __DEBUG__(a)
```

Comment:
I see you used a lot of debug point, is there a reason to add  `__DEBUG__(a)` instead of using the internal logger?
``` json
AddLog_P2(LOG_LEVEL_DEBUG, PSTR("any text here: %s or val here %d"), textstr, val);
```